src/bokeh/server/views/autoload_js_handler.py

```diff
@@ -89,6 +92,8 @@ async def get(self, *args, **kwargs):
 
     async def options(self, *args, **kwargs):
         '''Browsers make OPTIONS requests under the hood before a GET request'''
+        self.set_header('Access-Control-Allow-Methods', 'PUT, GET, OPTIONS')
+        self.set_header("Access-Control-Allow-Origin", self.request.headers["Origin"])
```

Comment:
For preflight requests cookies will never be sent. However, there is no downside in hardcoding the origin here. [`starlette`](https://github.com/encode/starlette) is doing the [same thing](https://github.com/encode/starlette/blob/4e453ce91940cc7c995e6c728e3fdf341c039056/starlette/middleware/cors.py#L114-L118).